data/datapackage.yaml

```diff
@@ -0,0 +1,338 @@
+        - name: quote_language_code
+          type: string
+          description: The ISO 639-1 codes for the book's text that was highlighted by the annotation
+        - name: annotator
+          type: string
+          description: The person who made the annotation # QUESTION: Add language about certainty? How did we know this?
+        - name: annotation_region
+          type: string
+          description: A link to the IIIF image of the book's page that has been cropped to show just the given annotation
+          constraints:
+            unique: true
+        - name: page_iiif
+          type: string
+          description: A link to the IIIF image of the book's page
+    scheme: file
+  - encoding: utf-8-sig
+    format: csv
+    hashing: md5
+    name: references
+    path: references.csv
+    description: References to other works made in Jacques Derrida's 1967 book De la grammatologie
+    profile: tabular-data-resource
+    schema:
+      foreignKeys:
+        - fields: book_id
+          reference: 
+            resource: library
+            fields: id
+      fields:
+        - name: id
+          type: string
+          description: An identifier linking to the reference's detail page. Note that these are not unique.
+          constraints:
+            required: true
+        - name: page
+          type: integer
+          description: The page in De la grammatologie where the reference was cited
+          constraints:
+            required: true
+        - name: page_location
+          type: string
+          description: '' # What is this? single letters are the paragraph?
+          constraints:
+            required: true
+        - name: type
+          type: string
+          description: How Derrida references the work (e.g. via a footnote, a quotation, etc.)
+          constraints:
+            required: true
+            pattern: "Footnote|Epigraph|Citation|Quotation"
+        - name: book_title
+          type: string
+          description: The title of the book referenced
+          constraints:
+            required: true
+        - name: book_id
+          type: string
+          format: uri
+          description: Link to the book where the reference was found
+          constraints:
+            required: true
+        - name: book_page
+          type: string
+          description: Page of the book in Derrida's citation
+        - name: book_type
+          type: string
+          description: Whether or not the reference is to a book or journal article
+          constraints:
+            required: true
+            pattern: Book|Journal Article
+        - name: anchor_text
+          type: string
+          description: The text of the quotation or the footnote that makes up the reference
+        - name: interventions
+          type: any
+          description: A URI to the linked annotation (if Derrida had a book where this reference was annotated.)
```

Comment:
this should be an annotation id, but I think very few of the connections were ever added to the database; do they match what ids in our annotation dataset?

Reply:
I confirmed that they're all in `annotations.csv`. Unfortunately we can't add this as an automatic check in the datapackage because they're multivalued.

Should I rename this to `annotations`?

Reply:
ah, good question... we should figure out how to be consistent in the data exports without diverging too much from the language of the project. Inclined to hold off on the rename until we get feedback from PRDS on our initial versions of data & metadata

Reply:
I will add this question to the document we share with Matt